main/midnight/src/tme/scenarios/ddr/ddr_processor_text.cpp

```diff
@@ -360,7 +360,7 @@ c_string tokens;
 
 std::string ddr_text::DescribeCharacterLocation( const mxcharacter* character )
 {
-    RETURN_IF_NULL(character);
+    RETURN_VALUE_IF_NULL(character, "");
```

Comment:
Corresponding use of macro.

Reply:
I don't think this is needed. The original **should** have read RETURN_IF_NULL(character) "";

Reply:
Makes sense. Making the change.

Reply:
Done.